### PR TITLE
LP-2908 - Add unit tests for Zoom client

### PR DESCRIPTION
**Pull request by aightmunam**
[LP-2908](https://philanthropyu.atlassian.net/browse/LP-2908)

***

#### Description:

- Add unit tests for the zoom client

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3054/workflows/workflows_id/jobs/3054/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/zoom_client/client.py (100%)
openedx/adg/common/lib/zoom_client/tests/conftest.py (100%)
openedx/adg/common/lib/zoom_client/tests/constants.py (100%)
openedx/adg/common/lib/zoom_client/tests/test_client.py (100%)
openedx/adg/conftest.py (100%)
-------------
Total:   140 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/common/lib/zoom_client/tests/test_client.py [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+            {
+                'type': AdgZoomClient.LICENSED_ACCOUNT_TYPE,
+                'account_id': DUMMY_ACCOUNT_ID,
+                'account_number': DUMMY_ACCOUNT_NUMBER - 1
+            }
+        ),
+        False
+    ),
+    (
+        (HTTP_200_OK, {'type': 1, 'account_id': DUMMY_ACCOUNT_ID, 'account_number': DUMMY_ACCOUNT_NUMBER}),
```
Comment (annas-usman) [inline openedx/adg/common/lib/zoom_client/tests/test_client.py:233]: Let's format this and break it into multiple lines as we have done for the above tuples for consistency.

**Review by annas-usman — APPROVED** [review]
LGTM, just a small suggestion

File: openedx/adg/common/lib/zoom_client/tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+"""
+All the fixtures for zoom client tests
+"""
+import json
+
+import pytest
+from requests.models import Response
+
+from openedx.adg.admin_dashboard.webinars.forms import WebinarForm
+from openedx.adg.common.lib.zoom_client.tests.constants import DUMMY_ZOOM_API_KEY
```
Comment (javeriaejaz-arbisoft) [inline openedx/adg/common/lib/zoom_client/tests/conftest.py:10]: lms/envs/test.py already have dummy values for keys.

**Pull request merged**